pkg/interface/chat/src/js/components/settings.js

```diff
@@ -171,15 +191,56 @@ export class SettingsScreen extends Component {
     if (!ownedUnmanagedVillage) {
       return null;
     } else {
+      let inclusiveToggle = <div/>
+      console.log('tg', state.targetGroup);
+      if (state.targetGroup) {
+        //TODO toggle component
```

Comment:
seems like it's here

Reply:
I meant like a `component/lib/toggle.js`. Will clarify.